cli/options/options.go

```diff
@@ -345,7 +367,10 @@ func GetAccFromContext(ctx *cli.Context) (*wallet.Account, *wallet.Wallet, error
 	if err != nil {
 		return nil, nil, err
 	}
-	addrFlag := ctx.Generic("address").(*flags.Address)
+	addrFlag, ok := ctx.Generic("address").(*flags.Address)
+	if !ok {
+		return nil, nil, errors.New("can't get address flag")
+	}
```

Comment:
How is it possible that conversion fails?

Reply:
Supposed we have no bugs in the flag code itself.

Reply:
left from debug. removed